opennms-doc/guide-admin/src/asciidoc/text/minion/jms.adoc

```diff
@@ -0,0 +1,81 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+=== Using JMS
```

Comment:
Note to self:
* Add section for authentication and authorization
* Add section for multi-tenancy (instance.id)